### ISSUE TITLE
Support /ping endpoint

### ISSUE THREAD

**Issue by shanab** [influxdata/influxdb-relay#24]
I'm using InfluxDB Relay to forward data to 2 InfluxDB instances + 1 Kapacitor instance. Eventually I would like kapacitor to write to Relay, the only problem now is that with the following kapacitor config:

```
[[influxdb]]
  enabled = true
  name = "localhost"
  default = true
  urls = ["http://192.168.50.11:9096", "http://192.168.50.12:9096"]
  disable-subscriptions = true
  ...
```

Kapacitor tries to ping a URL before it uses it, and if the ping fails then it assumes the URL is not working and tries the next one [(link to Kapacitor v0.13.1 client code)](https://github.com/influxdata/kapacitor/blob/v0.13.1/services/influxdb/service.go#L205).
So if I'm correct, kapacitor won't be able to write to InfluxDB Relay unless:
1. Relay supports /ping
2. Kapacitor doesn't use ping to determine if a client is working or not

My guess would be that (1) is cleaner?

**Comment by simcap** [comment]
Quite useful indeed. 

Recently, I kept getting some `invalid write endpoint` in production and was confused. My setup was going through relay to write to 2 InfluxDB instances. I had updated my components responsible to write to InfluxDB (and therefore going through relay) with a "on component startup" InfluxDB ping as a good practice.

Since relay did not accept `/ping` endpoint my components were getting the error `invalid write endpoint` comprehensibly.

Points writers to InfluxDB - even when going through relay - might want to ensure before writing that InfluxDB:
* is up and responding 
* get the time it took to responds and InfluxDB version (both included in the ping response)
